chunky/src/java/se/llbit/chunky/world/worldformat/Renderable.java

```diff
@@ -0,0 +1,10 @@
+package se.llbit.chunky.world.worldformat;
+
+import se.llbit.math.Octree;
+
+/** For simple world formats that only support being rendered directly */
+public interface Renderable extends Loadable {
+  void populateData(RenderableData data);
+
+  record RenderableData(Octree octree) {}
```

Comment:
That would require *two* octrees and a blockspec map, wouldn't it? As minimum data for rendering?

Reply:
Yes. The intention for this api is schematics where everything should be loaded immediately.

I guess if we're simulating regions they can just be smuggled in as real worlds too. Thoughts?

Reply:
Yes. The intention for this api is schematics where everything should be loaded immediately.

I guess if we're simulating regions they can just be smuggled in as real worlds too. Thoughts?

Reply:
Sounds good. Schematics support would be great! Maybe that's even be a good first test of this new API?